srml/contracts/src/lib.rs

```diff
@@ -229,6 +229,8 @@ where
 	}
 }
 
+// See FIXME #2744 (should result in removing this trait
```

Comment:
I don't like how this message is scattered in so many places. I'd leave it in one or two, maybe right above the actual accesses for any observant readers wondering about the indirection, but it doesn't need to be repeated everywhere TrieId is mentioned IMO.

Reply:
that was more a way to keep trace of every needed place to change, but obviously this will be resolved by the compilation.